app/src/test/java/com/github/geohunt/app/model/PointCalculatorTest.kt

```diff
@@ -0,0 +1,65 @@
+package com.github.geohunt.app.model
+
+import com.github.geohunt.app.model.database.api.Location
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+/**
+ * Tests for the PointCalculator
+ * Note that these tests are somewhat empiric
+ * and that changes to the hyper parameters of PointCalculator could break them
+ * pretty easily. Writing tests depending on these hyper parameters would be too
+ * complicated and not worth the time needed.
+ */
+class PointCalculatorTest {
+    @Test
+    fun calculatorGivesMaxPointsOnIdenticalLocation() {
+        val leukerbad = Location(46.37810088109651, 7.625535288545297)
+
+        assertEquals(PointCalculator.MAX_POINTS, PointCalculator.computePoints(leukerbad, leukerbad), 1e-3)
+    }
+
+    @Test
+    fun calculatorGivesZeroPointsOnLongDistances() {
+        //Distance of 2.75 km
+        val lausanne = Location(46.51919259139398, 6.633433265031245)
+        val pully = Location(46.50940191063235, 6.666380123092782)
+
+        assertEquals(PointCalculator.MIN_POINTS, PointCalculator.computePoints(lausanne, pully), 1e-3)
```

Comment:
Consider removing magic numbers in this class, such as 1e-3 and PointCalculator.MAX_POINTS as they lead to a code that is hard to read